services/QuillLMS/app/workers/admin_diagnostic_reports/diagnostic_overview_worker.rb

```diff
@@ -4,7 +4,7 @@ module AdminDiagnosticReports
   class DiagnosticOverviewWorker
     include Sidekiq::Worker
 
-    PUSHER_EVENT = 'admin-diagnostic-ovewview-cached'
+    PUSHER_EVENT = 'admin-diagnostic-overview-cached'
```

Comment:
@anathomical I noticed this misspelling-- is there anywhere else I need to worry about updating this value? (I already updated the `PUSHER_EVENT_KEY` in the `OverviewSection` frontend file)

Reply:
@eadams17 Good catch.  I think those should be the only two places where that value is defined: once on the front-end and once on the back-end.